src/test/java/collab/DeleteTest.java

```diff
@@ -28,6 +28,6 @@ public void deleteTest_returnEmployee() {
     public void deleteTest_NoReturn() {
       // "17112609","FB NTAWR","CL4","010-5645-6122","19860903","PRO"
       empDB.deleteItemByCondition("employeeNum", "17112609");
-      assertTrue(empDB.deleteItemByCondition("employeeNum", "17112609").size()==0);
+      assertEquals(empDB.deleteItemByCondition("employeeNum", "17112609").size(),0);
```

Comment:
assertEquals(기대값, 실제값)으로 쓰도록 정의되어 있네요.
실패했을 때 결과 표시도 Expected, Actual로 나오는 것으로 보아 이 형식에 맞추는 것이 좋을 것 같습니다.

```suggestion
      assertEquals(0, empDB.deleteItemByCondition("employeeNum", "17112609").size());
```